backend/src/src-moderation/src/main/java/org/eclipse/sw360/moderation/db/ClearingRequestRepository.java

```diff
@@ -94,7 +96,12 @@ public Set<ClearingRequest> getClearingRequestsByBU(String businessUnit) {
         return new HashSet<ClearingRequest>(queryView("byBusinessUnit", businessUnit));
     }
 
-    public Integer getCriticalClearingRequestCount() {
-        return new HashSet<ClearingRequest>(queryView("byPriority", ClearingRequestPriority.CRITICAL.name())).size();
+    public Integer getOpenCriticalClearingRequestCount(String group) {
+        Set<ClearingRequest> criticalCr = new HashSet<ClearingRequest>(queryView("byPriority", ClearingRequestPriority.CRITICAL.name()));
+        // filter the CLOSED / REJECTED and CR belong to same group as user
+        return CommonUtils.nullToEmptySet(criticalCr).stream()
```

Comment:
May be use `.distinct().count()` here after filter instead of collecting in Set and getting size.